lib/Dialect/SPIRV/SPIRVOps.cpp

```diff
@@ -953,6 +953,31 @@ static LogicalResult verify(spirv::BranchConditionalOp branchOp) {
 // spv.CompositeExtractOp
 //===----------------------------------------------------------------------===//
 
+static Type getElementType(Type type, ArrayAttr indices, Location loc) {
```

Comment:
Missing doc for this function.

"Walks walk the given `type` hierarchy with the given `indices`, potentially down to component granularity, to select an element type. Returns null type and emits errors with the given `loc` on failure."

Reply:
Thanks!